pyproject.toml

```diff
@@ -42,25 +42,23 @@ uproot = "*"
 importlib-metadata = {version = ">=4.8.1", python = "<3.8"}
 numba = "*"
 filelock = "*"
-
-[tool.poetry.dev-dependencies]
-Sphinx = "*"
-sphinx-rtd-theme = "*"
-numpydoc = "*"
-proposal = "7.6.2"
-pygdsm = {git = "https://github.com/telegraphic/pygdsm"}
-nifty5 = {git = "https://gitlab.mpcdf.mpg.de/ift/nifty.git", branch="NIFTy_5"}
-pypocketfft = {git = "https://gitlab.mpcdf.mpg.de/mtr/pypocketfft"}
-MCEq = "*"
-crflux = "*"
-pandas = "*"
-mattak = {git = "https://github.com/RNO-G/mattak"}
-runtable = {git = "ssh://git@github.com/RNO-G/rnog-runtable.git"}
+Sphinx = {version = "*", optional = true}
+sphinx-rtd-theme = {version = "*", optional = true}
+numpydoc = {version = "*", optional = true}
+proposal = {version = "7.6.2", optional = true}
+pygdsm = {git = "https://github.com/telegraphic/pygdsm", optional = true}
+nifty5 = {git = "https://gitlab.mpcdf.mpg.de/ift/nifty.git", branch="NIFTy_5", optional = true}
+pypocketfft = {git = "https://gitlab.mpcdf.mpg.de/mtr/pypocketfft", optional = true}
+MCEq = {version = "*", optional = true}
+crflux = {version = "*", optional = true}
+pandas = {version = "*", optional = true}
+mattak = {git = "https://github.com/RNO-G/mattak", optional = true}
+runtable = {git = "ssh://git@github.com/RNO-G/rnog-runtable.git", optional = true}
 
 [tool.poetry.extras]
 documentation = ["Sphinx", "sphinx-rtd-theme", "numpydoc"]
 proposal = ["proposal"]
 galacticnoise = ['pygdsm']
 ift_reco = ['nifty5', 'pypocketfft']
-muon_flux_calc = ['MCEq', 'crflux']
-RNO_G_DATA = ["mattak", "runtable", "pandas"]
```

Comment:
Why removing runtable and pandas from the RNO_G_DATA dependencies?

Reply:
I've added a separate `rno-g-extras` option now with the runtable, that does not get installed by default. Not sure if there is a better solution; in any case once you try to use the RNO-G reader it suggests installing the run table, I think.